packages/components/src/core/Button/index.stories.tsx

```diff
@@ -60,63 +60,199 @@ export const Default = {
   },
 };
 
-// Live Preview
+// Rounded Live Preview
 
-const styleLevel = {
+const placementStyles: React.CSSProperties = {
```

Comment:
Added back in the original Live Preview tests, but converted them to first create a "...Demo" component which is subsequently called by the main LivePreview components